templates/home.html

```diff
@@ -28,9 +28,9 @@ <h3 class="has-small-font-size" style="font-style:normal;font-weight:400;text-tr
 			<!-- wp:post-title {"level":4,"isLink":true,"style":{"spacing":{"margin":{"top":"var(--wp--preset--spacing--60)","bottom":"var(--wp--preset--spacing--60)"}}}} /-->
```

Comment:
Should these be `40` or something smaller to match the design more closely?

```suggestion
			<!-- wp:post-title {"level":4,"isLink":true,"style":{"spacing":{"margin":{"top":"var(--wp--preset--spacing--40)","bottom":"var(--wp--preset--spacing--40)"}}}} /-->
```

Reply:
I think 40 looks better in this case.